chart/values.yaml

```
@@ -43,6 +43,9 @@ controller:
       url: ""
       headersFilepath: ""
       headers: []
+      authURL: ""
+      authHeaders: ""
```

```suggestion
      authHeaders: []
```
if I'm not mistaken?